modules/gapi/src/backends/fluid/gfluidcore.cpp

```diff
@@ -2509,10 +2509,10 @@ GAPI_FLUID_KERNEL(GFluidSplit3, cv::gapi::core::GSplit3, false)
 
     static void run(const View &src, Buffer &dst1, Buffer &dst2, Buffer &dst3)
     {
-        const auto *in = src.InLine<uchar>(0);
-        auto *out1     = dst1.OutLine<uchar>();
-        auto *out2     = dst2.OutLine<uchar>();
-        auto *out3     = dst3.OutLine<uchar>();
+        const auto *in   = src.InLine<uchar>(0);
+              auto *out1 = dst1.OutLine<uchar>();
+              auto *out2 = dst2.OutLine<uchar>();
+              auto *out3 = dst3.OutLine<uchar>();
```

Comment:
There is no sense with aligning of `in` and `out` variables. They still stay unaligned. Just because they are from the different groups.

---

IMHO, we should not have unrelated changes in the patch. Code for review should be clear and should follow the purposes in PRs description.

Reply:
That PR is included in cascade of optimization PRs. In previous one https://github.com/opencv/opencv/pull/21520#discussion_r795683659 the alignment was changed and in this PR returned to be the same as in other kernels